sql/vecindex: TestDatadrivenVecIndex failed

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20330630?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20330630?buildTab=artifacts#/) on master @ [47eac7e2f4e1dd0a4e0998073631a0b5586e878b](https://github.com/cockroachdb/cockroach/commits/47eac7e2f4e1dd0a4e0998073631a0b5586e878b):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1193550971
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1193550971
--- FAIL: TestDatadrivenVecIndex (27.35s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-53458

Epic CRDB-21133

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20332674?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20332674?buildTab=artifacts#/) on master @ [34a67e113ae65b578b718690045806d2f07c6f9a](https://github.com/cockroachdb/cockroach/commits/34a67e113ae65b578b718690045806d2f07c6f9a):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3375465151
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3375465151
--- FAIL: TestDatadrivenVecIndex (23.91s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (8.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20336499?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20336499?buildTab=artifacts#/) on master @ [464489b6db1a8cf3a45727706f84f6a1afe97983](https://github.com/cockroachdb/cockroach/commits/464489b6db1a8cf3a45727706f84f6a1afe97983):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1339571536
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1339571536
--- FAIL: TestDatadrivenVecIndex (26.33s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20336953?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20336953?buildTab=artifacts#/) on master @ [1283b87ce2161807660263733659e3de15ff5faa](https://github.com/cockroachdb/cockroach/commits/1283b87ce2161807660263733659e3de15ff5faa):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3452105073
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3452105073
--- FAIL: TestDatadrivenVecIndex (22.24s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (5.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20342684?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20342684?buildTab=artifacts#/) on master @ [54bcb718d8801123e297b47c0b00dfb016fdc1a0](https://github.com/cockroachdb/cockroach/commits/54bcb718d8801123e297b47c0b00dfb016fdc1a0):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3128126914
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3128126914
--- FAIL: TestDatadrivenVecIndex (21.63s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (3.76s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20344236?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20344236?buildTab=artifacts#/) on master @ [f5c0bbdfd3cde3556340a67d7d910945ed961e99](https://github.com/cockroachdb/cockroach/commits/f5c0bbdfd3cde3556340a67d7d910945ed961e99):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2983450397
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2983450397
--- FAIL: TestDatadrivenVecIndex (27.36s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.17s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20350273?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20350273?buildTab=artifacts#/) on master @ [beecf575f088f8036bc2cb78f9d277cd06d604d8](https://github.com/cockroachdb/cockroach/commits/beecf575f088f8036bc2cb78f9d277cd06d604d8):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1049351923
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1049351923
--- FAIL: TestDatadrivenVecIndex (36.95s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (10.25s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20351903?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20351903?buildTab=artifacts#/) on master @ [8f5ae52a8f9a419efd0c4ab26464ff9b265ad1f3](https://github.com/cockroachdb/cockroach/commits/8f5ae52a8f9a419efd0c4ab26464ff9b265ad1f3):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3599048998
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3599048998
--- FAIL: TestDatadrivenVecIndex (18.49s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (3.79s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20358160?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20358160?buildTab=artifacts#/) on master @ [fdb56245f7d6367aa1db9adccf80540809998f74](https://github.com/cockroachdb/cockroach/commits/fdb56245f7d6367aa1db9adccf80540809998f74):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex748158924
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex748158924
--- FAIL: TestDatadrivenVecIndex (24.19s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20359847?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20359847?buildTab=artifacts#/) on master @ [ae83e7ace8b574f37cd897aa838233e6f14ede96](https://github.com/cockroachdb/cockroach/commits/ae83e7ace8b574f37cd897aa838233e6f14ede96):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4040640418
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4040640418
--- FAIL: TestDatadrivenVecIndex (34.33s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (9.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20365717?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20365717?buildTab=artifacts#/) on master @ [7c9d33444528537a773ac721ad2b1b6318b413a9](https://github.com/cockroachdb/cockroach/commits/7c9d33444528537a773ac721ad2b1b6318b413a9):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1824416976
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1824416976
--- FAIL: TestDatadrivenVecIndex (30.77s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.84s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20366917?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20366917?buildTab=artifacts#/) on master @ [69f7088d7a199c11b4a27cc643ebae11f6e310dc](https://github.com/cockroachdb/cockroach/commits/69f7088d7a199c11b4a27cc643ebae11f6e310dc):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2805820356
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2805820356
--- FAIL: TestDatadrivenVecIndex (26.50s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (3.45s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20369462?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20369462?buildTab=artifacts#/) on master @ [484d0e454b6e01b08bf5a7ed5d3d95e1c2cab622](https://github.com/cockroachdb/cockroach/commits/484d0e454b6e01b08bf5a7ed5d3d95e1c2cab622):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3170141032
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3170141032
--- FAIL: TestDatadrivenVecIndex (30.01s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.20s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20373252?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20373252?buildTab=artifacts#/) on master @ [431cd74b1615636f16515a90f346fc65209df4b1](https://github.com/cockroachdb/cockroach/commits/431cd74b1615636f16515a90f346fc65209df4b1):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2449553086
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2449553086
--- FAIL: TestDatadrivenVecIndex (29.38s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.35s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20376504?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20376504?buildTab=artifacts#/) on master @ [82759997da526f08ead6de93e47fb30819edf10e](https://github.com/cockroachdb/cockroach/commits/82759997da526f08ead6de93e47fb30819edf10e):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1989666229
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1989666229
--- FAIL: TestDatadrivenVecIndex (28.66s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.07s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20383249?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20383249?buildTab=artifacts#/) on master @ [f5f0a4a0b27f9af0a3529d9321027dc1690136b2](https://github.com/cockroachdb/cockroach/commits/f5f0a4a0b27f9af0a3529d9321027dc1690136b2):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2191947049
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2191947049
--- FAIL: TestDatadrivenVecIndex (19.16s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (3.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20383405?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20383405?buildTab=artifacts#/) on master @ [9e83fa37459541594a9bb499f7cc18c155e37538](https://github.com/cockroachdb/cockroach/commits/9e83fa37459541594a9bb499f7cc18c155e37538):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3265755543
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3265755543
--- FAIL: TestDatadrivenVecIndex (29.61s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (8.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20384099?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20384099?buildTab=artifacts#/) on master @ [afdad220750ae62bc9dafbe6ca9d22ab4941095e](https://github.com/cockroachdb/cockroach/commits/afdad220750ae62bc9dafbe6ca9d22ab4941095e):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4276879166
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4276879166
--- FAIL: TestDatadrivenVecIndex (32.67s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.53s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20393395?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20393395?buildTab=artifacts#/) on master @ [3f23b10103e1d75b6868da67acf5211a73e82402](https://github.com/cockroachdb/cockroach/commits/3f23b10103e1d75b6868da67acf5211a73e82402):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3790342693
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3790342693
--- FAIL: TestDatadrivenVecIndex (25.94s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.17s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20394647?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20394647?buildTab=artifacts#/) on master @ [2f096411ae050af2ed461703e3ed1214ba097e56](https://github.com/cockroachdb/cockroach/commits/2f096411ae050af2ed461703e3ed1214ba097e56):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1358467835
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1358467835
--- FAIL: TestDatadrivenVecIndex (31.62s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.41s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20396676?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20396676?buildTab=artifacts#/) on master @ [9e30767e7a4c6b3e9d8232a2edbd0dc2d835f34c](https://github.com/cockroachdb/cockroach/commits/9e30767e7a4c6b3e9d8232a2edbd0dc2d835f34c):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4129645655
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4129645655
--- FAIL: TestDatadrivenVecIndex (34.78s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.69s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20398156?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20398156?buildTab=artifacts#/) on master @ [3d5ef9464d282f18aaab842d2dc0bd5d49353f41](https://github.com/cockroachdb/cockroach/commits/3d5ef9464d282f18aaab842d2dc0bd5d49353f41):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4166857361
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4166857361
--- FAIL: TestDatadrivenVecIndex (26.68s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (8.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20402160?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20402160?buildTab=artifacts#/) on master @ [91b3eee0f04549bf73bf1d35cc2da8a258ff5324](https://github.com/cockroachdb/cockroach/commits/91b3eee0f04549bf73bf1d35cc2da8a258ff5324):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex93197632
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex93197632
--- FAIL: TestDatadrivenVecIndex (33.73s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.21s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20403953?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20403953?buildTab=artifacts#/) on master @ [a5cca5e82f985ef04f93fc87b70356dff10957e9](https://github.com/cockroachdb/cockroach/commits/a5cca5e82f985ef04f93fc87b70356dff10957e9):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex211045933
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex211045933
--- FAIL: TestDatadrivenVecIndex (24.38s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (3.81s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20406676?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20406676?buildTab=artifacts#/) on master @ [fbc73a7cc20ef29891787ff5c3b30c36b150f908](https://github.com/cockroachdb/cockroach/commits/fbc73a7cc20ef29891787ff5c3b30c36b150f908):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2703827481
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2703827481
--- FAIL: TestDatadrivenVecIndex (27.15s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20408897?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20408897?buildTab=artifacts#/) on master @ [921d8185301766b0f50600c7373e0dd8e105b215](https://github.com/cockroachdb/cockroach/commits/921d8185301766b0f50600c7373e0dd8e105b215):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2180686437
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
--- FAIL: TestDatadrivenVecIndex (33.12s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20409652?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20409652?buildTab=artifacts#/) on master @ [e676b749a5a8d79448c588f72faea85b35412e6b](https://github.com/cockroachdb/cockroach/commits/e676b749a5a8d79448c588f72faea85b35412e6b):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2659452161
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2659452161
--- FAIL: TestDatadrivenVecIndex (22.33s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.17s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20410745?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20410745?buildTab=artifacts#/) on master @ [de1bfff38dddf6fbeb1d6958bdd2bd513e572e24](https://github.com/cockroachdb/cockroach/commits/de1bfff38dddf6fbeb1d6958bdd2bd513e572e24):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1868636288
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1868636288
--- FAIL: TestDatadrivenVecIndex (22.98s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20411740?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20411740?buildTab=artifacts#/) on master @ [e3770b8d2034d7eeaa5095a3b2214da3f35b895e](https://github.com/cockroachdb/cockroach/commits/e3770b8d2034d7eeaa5095a3b2214da3f35b895e):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1282614553
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1282614553
--- FAIL: TestDatadrivenVecIndex (24.55s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.83s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20413172?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20413172?buildTab=artifacts#/) on master @ [0c85e9dc66ab1df7db1cb1407c2a470289be61b8](https://github.com/cockroachdb/cockroach/commits/0c85e9dc66ab1df7db1cb1407c2a470289be61b8):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3715894929
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3715894929
--- FAIL: TestDatadrivenVecIndex (31.65s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (9.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20414573?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20414573?buildTab=artifacts#/) on master @ [ea6772d06fc62b05b4b6e72602853a8e591bfd77](https://github.com/cockroachdb/cockroach/commits/ea6772d06fc62b05b4b6e72602853a8e591bfd77):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2054109554
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2054109554
--- FAIL: TestDatadrivenVecIndex (35.10s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.86s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20419433?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20419433?buildTab=artifacts#/) on master @ [9a77ab62090722887cc70dc1d33c1345a2d675a6](https://github.com/cockroachdb/cockroach/commits/9a77ab62090722887cc70dc1d33c1345a2d675a6):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3712893569
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3712893569
--- FAIL: TestDatadrivenVecIndex (32.03s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20422488?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20422488?buildTab=artifacts#/) on master @ [5039022679074947f9988e76ac3870724d0f0f42](https://github.com/cockroachdb/cockroach/commits/5039022679074947f9988e76ac3870724d0f0f42):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2473202979
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2473202979
--- FAIL: TestDatadrivenVecIndex (25.77s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.72s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20424352?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20424352?buildTab=artifacts#/) on master @ [7b65b179304f8667c4cbd8d949208ba273195fb9](https://github.com/cockroachdb/cockroach/commits/7b65b179304f8667c4cbd8d949208ba273195fb9):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1837359982
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1837359982
--- FAIL: TestDatadrivenVecIndex (43.08s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (9.39s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20427240?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20427240?buildTab=artifacts#/) on master @ [eb8c0d98da22725422f12eac0f757772de9f0e00](https://github.com/cockroachdb/cockroach/commits/eb8c0d98da22725422f12eac0f757772de9f0e00):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3608169262
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3608169262
--- FAIL: TestDatadrivenVecIndex (28.43s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.71s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20429054?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20429054?buildTab=artifacts#/) on master @ [540d64551d7b1773fdadd79ddf7fb7fbd9605bc0](https://github.com/cockroachdb/cockroach/commits/540d64551d7b1773fdadd79ddf7fb7fbd9605bc0):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1853361473
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1853361473
--- FAIL: TestDatadrivenVecIndex (36.52s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20433115?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20433115?buildTab=artifacts#/) on master @ [fa76ba12e60927d8aa5ebad0eb64221f121927de](https://github.com/cockroachdb/cockroach/commits/fa76ba12e60927d8aa5ebad0eb64221f121927de):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3599350023
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3599350023
--- FAIL: TestDatadrivenVecIndex (32.22s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (10.69s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20437027?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20437027?buildTab=artifacts#/) on master @ [3fa36dceb664e5fe1fc0c8fb0253e7331124fc96](https://github.com/cockroachdb/cockroach/commits/3fa36dceb664e5fe1fc0c8fb0253e7331124fc96):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4265643259
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4265643259
--- FAIL: TestDatadrivenVecIndex (32.70s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (8.71s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20438663?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20438663?buildTab=artifacts#/) on master @ [e77f882443217c3271009077fcecd4feee86751f](https://github.com/cockroachdb/cockroach/commits/e77f882443217c3271009077fcecd4feee86751f):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex709518360
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex709518360
--- FAIL: TestDatadrivenVecIndex (33.23s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.97s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20438764?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20438764?buildTab=artifacts#/) on master @ [b8197d505b173f1e44e68f72a8bb679959923210](https://github.com/cockroachdb/cockroach/commits/b8197d505b173f1e44e68f72a8bb679959923210):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1458726878
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1458726878
--- FAIL: TestDatadrivenVecIndex (50.33s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (11.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20439612?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20439612?buildTab=artifacts#/) on master @ [8b1e1a9e399ca84a87c686d0106279e0c5904871](https://github.com/cockroachdb/cockroach/commits/8b1e1a9e399ca84a87c686d0106279e0c5904871):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2232563558
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2232563558
--- FAIL: TestDatadrivenVecIndex (30.43s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.84s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20446561?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20446561?buildTab=artifacts#/) on master @ [ded3a0d34ec27476dc9b4e3554fdd9f7555d8781](https://github.com/cockroachdb/cockroach/commits/ded3a0d34ec27476dc9b4e3554fdd9f7555d8781):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2052211988
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2052211988
--- FAIL: TestDatadrivenVecIndex (26.68s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.29s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20447906?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20447906?buildTab=artifacts#/) on master @ [e014575819f9df5e23d75a42e9341e1d63ab5016](https://github.com/cockroachdb/cockroach/commits/e014575819f9df5e23d75a42e9341e1d63ab5016):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4102944113
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex4102944113
--- FAIL: TestDatadrivenVecIndex (29.82s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (7.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20447962?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20447962?buildTab=artifacts#/) on master @ [dec7e21fb15915e6cea4498458b64656ddc95a49](https://github.com/cockroachdb/cockroach/commits/dec7e21fb15915e6cea4498458b64656ddc95a49):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3294811775
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3294811775
--- FAIL: TestDatadrivenVecIndex (32.89s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (5.97s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20449525?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20449525?buildTab=artifacts#/) on master @ [7073d3e4886b7414ed04172406f9b9c4f44dabda](https://github.com/cockroachdb/cockroach/commits/7073d3e4886b7414ed04172406f9b9c4f44dabda):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3526365222
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3526365222
--- FAIL: TestDatadrivenVecIndex (43.05s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (10.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20464371?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20464371?buildTab=artifacts#/) on master @ [7fdf860fb3b5b3d20a4e0f52e19ce06d256ab0f6](https://github.com/cockroachdb/cockroach/commits/7fdf860fb3b5b3d20a4e0f52e19ce06d256ab0f6):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex590528928
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex590528928
--- FAIL: TestDatadrivenVecIndex (39.50s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (8.42s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20465336?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20465336?buildTab=artifacts#/) on master @ [fe5ec41bce97958b4904bf898beddfe43d3f4e13](https://github.com/cockroachdb/cockroach/commits/fe5ec41bce97958b4904bf898beddfe43d3f4e13):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1953792274
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex1953792274
--- FAIL: TestDatadrivenVecIndex (30.56s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.47s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20466582?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20466582?buildTab=artifacts#/) on master @ [2b316d4d62d7695aa3ef27e00f81b9f69e3d1084](https://github.com/cockroachdb/cockroach/commits/2b316d4d62d7695aa3ef27e00f81b9f69e3d1084):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3047758519
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3047758519
--- FAIL: TestDatadrivenVecIndex (22.45s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (4.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20466798?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20466798?buildTab=artifacts#/) on master @ [8b39430fe2e0760df13d12a2c95e91600aff4efe](https://github.com/cockroachdb/cockroach/commits/8b39430fe2e0760df13d12a2c95e91600aff4efe):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3325323787
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex3325323787
--- FAIL: TestDatadrivenVecIndex (34.33s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (9.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20468589?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20468589?buildTab=artifacts#/) on master @ [90db6eab29659bc86e8ce6d6314f559e99f1068d](https://github.com/cockroachdb/cockroach/commits/90db6eab29659bc86e8ce6d6314f559e99f1068d):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2788943942
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2788943942
--- FAIL: TestDatadrivenVecIndex (30.50s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (6.85s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/vecindex.TestDatadrivenVecIndex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20469144?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20469144?buildTab=artifacts#/) on master @ [e4db2dd74b36adb8420252e30065a937406edaac](https://github.com/cockroachdb/cockroach/commits/e4db2dd74b36adb8420252e30065a937406edaac):


```
=== RUN   TestDatadrivenVecIndex
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2176405599
    test_log_scope.go:82: use -show-logs to present logs inline
    vecindex_test.go:90: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/4154236b1ebfe2c1ed8aa545a91a79fa/logTestDatadrivenVecIndex2176405599
--- FAIL: TestDatadrivenVecIndex (34.59s)
=== RUN   TestDatadrivenVecIndex/images-512d-euclidean
    test_server_shim.go:154: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:5:
          new-index dataset=images-512d-10k.gob train-count=1000 min-partition-size=4 max-partition-size=16 quality-samples=8 beam-size=4
          ----
          Created index with 1000 vectors with 512 dimensions.
    vecindex_test.go:71: 
        testdata/images-512d-euclidean.ddt:9:
          recall topk=10 beam-size=1 samples=20
        
        expected:
          24.00% recall@10
        
        found:
          20.00% recall@10
--- FAIL: TestDatadrivenVecIndex/images-512d-euclidean (10.28s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestDatadrivenVecIndex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/vecindex/vecindex_test.go
+++ b/pkg/sql/vecindex/vecindex_test.go
@@ -45,6 +45,10 @@ func TestDatadrivenVecIndex(t *testing.T) {
 
 	skip.UnderRace(t, "test is too slow under race")
 
+	// Skip on s390x due to floating point precision differences.
+	// See: https://github.com/cockroachdb/cockroach/issues/151766
+	skip.OnArch(t, "s390x", "data driven vector index tests disabled on s390x due to issue #151766")
+
 	datadriven.Walk(t, "testdata", func(t *testing.T, path string) {
 		if !strings.HasSuffix(path, ".ddt") {
 			// Skip files that are not data-driven tests.

--- a/pkg/testutils/skip/skip.go
+++ b/pkg/testutils/skip/skip.go
@@ -9,6 +9,7 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"runtime"
 	"strings"
 	"testing"
 
@@ -212,6 +213,14 @@ func UnderRemoteExecutionWithIssue(t SkippableTest, githubIssueID int, args ...i
 		maybeSkip(t, withIssue("disabled under race", githubIssueID), args...)
 	}
 
+}
+
+func OnArch(t SkippableTest, arch string, args ...interface{}) {
+	t.Helper()
+	if runtime.GOARCH == arch {
+		skipReason := fmt.Sprintf("on-arch %s (current config %s)", arch, testConfig())
+		maybeSkip(t, skipReason, args...)
+	}
 }
 
 func testConfig() string {

